src/App.js

```diff
@@ -1,16 +1,19 @@
+/* eslint-disable import/no-extraneous-dependencies */
 import './App.css';
 // eslint-disable-next-line import/no-extraneous-dependencies
 import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
 import Navbar from './components/Navbar';
-import BookList from './components/BookList';
+// import BookList from './components/BookList';
+import BooksContainer from './components/pages.js/BooksContainer';
 
 function App() {
+  // const dispatch = useDispatch();
```

Comment:
- Kindly remove this commented piece of code, it makes your code not maintainable and clean.